apps/rocketchat/template-rocketchat-mongodb.yaml

```diff
@@ -19,6 +19,7 @@ parameters:
     displayName: File Upload Storage Size
```

Comment:
Sorry I'm a jerk and didn't push my changes early enough. https://github.com/BCDevOps/platform-services/tree/rc-work/apps/rocketchat Can you rebase and and you updates in. There is now 3 template files. 1 for mongo. 1 for rocket chat. 1 for mongobackup

Reply:
haha no worries, now I know where to put things as well. will rebase.